legend-engine-xt-relationalStore-executionPlan-connection-api/src/main/java/org/finos/legend/engine/plan/execution/stores/relational/connection/api/schema/SchemaExplorationApi.java

```diff
@@ -76,4 +75,20 @@ public Response buildDatabase(DatabaseBuilderInput databaseBuilderInput, @Pac4JP
         }
     }
 
+    @Path("adhocExecuteSQL")
```

Comment:
why not just `executeSQL` ?

Reply:
I felt like this flow is still very much experimental and I don't want to give it a name that might rival the popularity of the mainstream flow such as using our PostgreSQL JDBC

Reply:
Renamed to `executeRawSQL`